cfgov/jinja2/v1/_includes/molecules/expandable.html

```diff
@@ -40,12 +40,11 @@
     %}
 {% endif %}
 {% set value = value or global_dict.expandable %}
-
 <div class="m-expandable
-            m-expandable__expanded
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/4b62b60e5fbeb77cf6c5cc4f29c2bc12f09a272c